auto_submit/bin/server.dart

```diff
@@ -48,6 +49,13 @@ Future<void> main() async {
         ReadinessCheck(
           config: config,
         ).run,
+      )
+      ..get(
+        '/update-revert-issues',
+        UpdateRevertReviews(
```

Comment:
Can we update the class name accordingly, also the file name?

Reply:
Yeah you got it!